cmake/std/atdm/cts1empire/environment_new.sh

```diff
@@ -35,8 +35,15 @@ fi
 sparc_tpl_base=${ATDM_CONFIG_SPARC_TPL_BASE}
 
 if [ "$ATDM_CONFIG_COMPILER" == "INTEL-18.0.2_OPENMPI-4.0.1" ]; then
```

Comment:
Obviously, this line is now a lie, since I was having trouble passing the right variables through to the build otherwise.